pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -38,7 +38,7 @@
 , pdal
 , zstd
 , makeWrapper
-, wrapGAppsHook
```

Comment:
Did you test that `wrapGAppsHook` is really not needed on non-NixOS platforms ?

Reply:
No I think I should probably revert this.